SQL/New_patches/2018-11-13_add_path_to_config_enum.sql

```diff
@@ -4,5 +4,5 @@
 -- will be validated when configured from the front-end and will throw errors
 -- if not reachable by the server.
 ALTER TABLE ConfigSettings MODIFY COLUMN DataType enum('text','boolean','email','instrument','textarea','scan_type','lookup_center','path','web_path');
-UPDATE ConfigSettings SET DataType='web_path' where Parent=24;
+UPDATE ConfigSettings SET DataType='web_path' where Name IN ('imagePath', 'base', 'data', 'extLibs', 'mincPath', 'DownloadPath', 'IncomingPath', 'MRICodePath', 'MRIUploadIncomingPath', 'GenomicDataPath', 'mediaPath');
```

Comment:
@cmadjar @davidblader @nicolasbrossard can any of you comment on if any of these are used by the imaging pipeline/need to be relative to the imaging server and not the web server?

Reply:
I'll bring this up in a LORIS meeting since the discussion seems to be stalled.

Reply:
@johnsaigle so sorry, for some reason I did not see that thread. 

### web-path:
- `data`
- `mincPath`
- `GenomicDataPath`
- `imagePath`
- `mediaPath`
- `DownloadPath`
- `log`
- `base`
- `extLibs` I guess? No idea what this one is suppose to be but it is not imaging related so I guess it is web-path

### path:

- `MRICodePath`
- `tarchiveLibraryDir`
- `get_dicom_info`
- `niak_path`

### both path and web-path in my head

- `MRIUploadIncomingPath` (used by the imaging uploader to upload data on the server in the path mentioned)
- `dataDirBasepath` (used by the imaging scripts but will eventually be used by the LORIS modules as well so that we do not have 4 config options that refers to the same thing, aka `imagePath` = `mincPath` = `data` = `dataDirBasepath`, see https://github.com/aces/Loris/issues/4197)

Hope this helps!! Let me know if you need more info from me.

Reply:
@cmadjar Thanks so much!

Reply:
@cmadjar I don't see how the "both path and web-path" ones are (going to/) work.. what happens if they're mounted at different places on the web server and on the processing server?